coderd/database/migrations/000258_add_otp_reset_to_user.down.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE users DROP COLUMN hashed_one_time_passcode;
```

Comment:
Did you consider using a token instead of a passcode? Either works I guess but I token is more common.

Reply:
@johnstcn @dannykopping thoughts on token vs passcode? I'm neutral either way.

Reply:
Let's stick with what you have for now.